EnglishTranslation-master/ETC.tsv

```diff
@@ -4265,7 +4265,7 @@ ETC_20150317_004264	Fool (haha)
 ETC_20150317_004265	The amount is low.
 ETC_20150317_004266	Moni
 ETC_20150317_004267	 key has been assigned to 
-ETC_20150317_004268	 to your quick slot.
+ETC_20150317_004268	 on your quick slot bar.
```

Comment:
Am I missing something? The text 'Assign to' is present and not changed. This change would read as "(keyname) key has been assigned to (skill name) on your quick slot bar."